src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -461,4 +463,17 @@ protected function isGrantedOr403($permission)
             throw new AccessDeniedHttpException();
         }
     }
+
+    protected function validateDependencies()
```

Comment:
I prefer something more generic which will check if a bundle is registered or not.